src/CodeGen_Hexagon.cpp

```diff
@@ -54,10 +54,6 @@ class CodeGen_Hexagon : public CodeGen_Posix {
                                          std::vector<Type> arg_types,
                                          int flags);
 
-    int is_hvx_v65_or_later() const {
```

Comment:
@aankit-ca  - where is ``hvx_v66`` needed? I dont see any use of anything that is available at v66 but not v65, but I may be mistaken.